.gitignore

```diff
@@ -6,3 +6,5 @@ dev-env/unittest/bin
 
 # generated by helm chart-releaser
 .cr-*
+
+.idea/
```

Comment:
please add this as a separate commit

Reply:
ok

Reply:
note that this doesn't necessarily have to be a separate pr, but just a separate commit in the same pr for trivial things such as this is fine.